### ISSUE TITLE
Keyboard navigation does not work on submenus after first open

### ISSUE THREAD

**Issue by shanebeehler** [vidstack/player#1088]
### Current Behavior:

When navigating with keyboard, menus with submenus are navigational only on first use. If you close out, next time you navigate with keyboard, the tabindex is set to -1 and is not navigational with keyboard. This does not happen on menus without submenus.

### Expected Behavior:

Expect that subsequent uses of menus with submenus are accessible via keyboard navigation.

### Steps To Reproduce:

https://stackblitz.com/edit/vidstack-examples-hcphp2?file=src%2Fplayer.tsx&showSidebar=1&title=Vidstack%20Player%20-%20React%20(Default%20Layout)

You can see on the provided Stackblitz that is from the example repo. Notice that the chapters menu keyboard navigation work after several uses, but the settings menu keyboard navigation stops working after the first use. Only a hard refresh fixes.
